src/store/flow/block-types/ConsoleIO_PrintBlockStore.ts

```diff
@@ -50,6 +51,9 @@ export const actions: ActionTree<IFlowsState, IRootState> = {
     }, {root: true})
   },
 
+  validate_block({rootGetters}, {block, schemaVersion}: {block: IBlock, schemaVersion: string}) {
```

Comment:
I think the `validate_block` action should be `async`, right ? (According to ACs)

Reply:
2nd thought:
I'm wondering if we can rename this to `validate( )` only. In fact, it's already under the block-type module.
Then in validation module, when we call `validate_block()`, we will dispatch `dispatch('flow/${block.type}/validate'` instead of `dispatch('flow/${block.type}/validate_block'`. Which is less confusing for me.

Reply:
I agree with naming this method validate as it is inside the respective block stores.

Reply:
Regarding `async` for `validate` method, I feel it may not be necessary as we are awaiting the dispatched action to finish in our `validation/index.ts validate_block` method as seen below. We are not having an asynchronous operation inside the validate method itself. 
Let me know if I am missing something?   I would be happy to explain or provide more clarity on what I was thinking here.

Reply:
Thanks for clarifying @poojahpatil90 , ... let's move it to QA then. I'm still a bit confused about the `async` but I think it's ok as long as it's working :D